### PR TITLE
Added description to help understand challenge

### PR DESCRIPTION
**Pull request by adam-weiler**
I added a short description at the beginning to explain that the user needs to use a different starter project than the one they were using earlier.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/understand-bcrypt-hashes.english.md [hunk 1]
```diff
@@ -6,11 +6,12 @@ challengeType: 2
 
 ## Description
 <section id='description'>
-As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-bcrypt/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-bcrypt/'>GitHub</a>.
+For the following challenges, we are going to be working on a new starter project that is different from earlier challenges. This project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-bcrypt/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-bcrypt/'>GitHub</a>.
```
Comment (RandellDawson) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/understand-bcrypt-hashes.english.md]:
```suggestion
For the following challenges, you will be working with a new starter project that is different from earlier challenges. This project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-bcrypt/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-bcrypt/'>GitHub</a>.
```

File: curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/understand-bcrypt-hashes.english.md [hunk 2]
```diff
@@ -6,11 +6,12 @@ challengeType: 2
...
 <hr>To begin using BCrypt, add it as a dependency in your project and require it as 'bcrypt' in your server.
 Submit your page when you think you've got it right.
+Remember, the tests will not pass unless you are working on the new starter project.
```
Comment (RandellDawson) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/understand-bcrypt-hashes.english.md]: I think once at the top is enough.
```suggestion

```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
Please see my suggested changes for this PR.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/understand-bcrypt-hashes.english.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ challengeType: 2
 
 ## Description
 <section id='description'>
-As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-bcrypt/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-bcrypt/'>GitHub</a>.
+For the following challenges, you will be working with a new starter project that is different from earlier challenges. This project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-bcrypt/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-bcrypt/'>GitHub</a>.
```
Comment (RandellDawson) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/understand-bcrypt-hashes.english.md]: @adam-weiler Quick question: Should it be `For the following challenge` or `For the following challenges`?

Comment (adam-weiler) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/understand-bcrypt-hashes.english.md]: This starter project is used for lessons 12, 13, and 14 of the unit:
-Understand BCrypt Hashes
-Hash and Compare Passwords Asynchronously
-Hash and Compare Passwords Synchronously

Lessons 1-11 of the "Information Security with HelmetJS" unit use a different starter project. That's why I thought it is important to point out the difference, so students realize to fork the new starter.

Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/understand-bcrypt-hashes.english.md]: OK, just confirming.

**Review by RandellDawson — APPROVED** [review]
LGTM

**Review by cmccormack — APPROVED** [review]
This looks good to me 👍.  Thank you for your contribution!

**Pull request merged**